### PR TITLE
[NUI] remove "_" and refactoring naming to pascal case.

### PR DESCRIPTION
**Pull request by JoogabYun**
### Description of Change ###

 1. Remove class name from api name 
  : View_HasKeyInputFocus() -> HasKeyInputFocus() in Interop.View class
 2. Remove "_" and change to Pascal Case
  : TextEditor_Property_TEXT_COLOR_get();  -> TextColorGet();
 3. Remove "__SWIG_" suffix
  : new_TextEditor__SWIG_0(); -> NewTextEditor();

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1707

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/internal/Application.cs [hunk 1]
```diff
@@ -1342,70 +1342,70 @@ public static List<Window> GetWindowList()
 
         internal ApplicationSignal InitSignal()
         {
-            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.Application_InitSignal(swigCPtr), false);
+            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.ApplicationInitsignal(swigCPtr), false);
```
Comment (m-macias) [inline src/Tizen.NUI/src/internal/Application.cs:1345]:
```suggestion
            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.ApplicationInitSignal(swigCPtr), false);
```

File: src/Tizen.NUI/src/internal/Application.cs [hunk 2]
```diff
@@ -1342,70 +1342,70 @@ public static List<Window> GetWindowList()
...
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
 
         internal ApplicationSignal TerminateSignal()
         {
-            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.Application_TerminateSignal(swigCPtr), false);
+            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.ApplicationTerminatesignal(swigCPtr), false);
```
Comment (m-macias) [inline src/Tizen.NUI/src/internal/Application.cs:1352]:
```suggestion
            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.ApplicationTerminateSignal(swigCPtr), false);
```

File: src/Tizen.NUI/src/internal/Application.cs [hunk 3]
```diff
@@ -1342,70 +1342,70 @@ public static List<Window> GetWindowList()
...
-            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.Application_RegionChangedSignal(swigCPtr), false);
+            ApplicationSignal ret = new ApplicationSignal(NDalicPINVOKE.ApplicationRegionchangedsignal(swigCPtr), false);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
 
         internal LowBatterySignalType BatteryLowSignal()
         {
-            LowBatterySignalType ret = new LowBatterySignalType(NDalicPINVOKE.Application_LowBatterySignal(swigCPtr), false);
+            LowBatterySignalType ret = new LowBatterySignalType(NDalicPINVOKE.ApplicationLowbatterysignal(swigCPtr), false);
```
Comment (m-macias) [inline src/Tizen.NUI/src/internal/Application.cs:1401]:
```suggestion
            LowBatterySignalType ret = new LowBatterySignalType(NDalicPINVOKE.ApplicationLowBatterySignal(swigCPtr), false);
```

File: src/Tizen.NUI/src/internal/FixedRuler.cs [hunk 4]
```diff
@@ -72,7 +72,7 @@ public override float GetPositionFromPage(uint page, out uint volume, bool wrap)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override uint GetPageFromPosition(float position, bool wrap)
         {
-            uint ret = Interop.Ruler.FixedRuler_GetPageFromPosition(swigCPtr, position, wrap);
+            uint ret = Interop.Ruler.FixedrulerGetpagefromposition(swigCPtr, position, wrap);
```
Comment (m-macias) [inline src/Tizen.NUI/src/internal/FixedRuler.cs:75]:
```suggestion
            uint ret = Interop.Ruler.FixedRulerGetPageFromPosition(swigCPtr, position, wrap);
```

**Review by m-macias — COMMENTED** [review]
Maybe there is a name collision and it is a reason why it was named that way, but there are multiple instances that use PascalCase incorrectly. I marked only some of them.

**Comment by dongsug-song** [comment]
@JoogabYun 
Hello, 
Thank you for your huge job done. 
but as you know, it need get success from checker
please check errors.
thank you.

**Comment by JoogabYun** [comment]
@m-macias 
Thank you for your review.
I will check again for the pascal case transformation.

**Comment by dongsug-song** [comment]
@JoogabYun 
Hello, 
thank you for your big work.
but could you resolve conflicts?
and also could you get @m-macias 's review again?
thank you.

**Comment by JoogabYun** [comment]
@dongsug-song 
I updated patch. and I took care of m-macias's review.
Fixed incorrect Pascal case
ex) ApplicationLowBatterysignal -> ApplicationLowBatterySignal

**Review by dongsug-song — APPROVED** [review]
I think there would be build errors in GBM's image release. 
for that case, I will request for you to fix those errors by adding same APIs which just have different names.
in parrallel we will make some list of these APIs for GBM to change.
thank you.

**Pull request merged**